src/containers/VerifyActivities.jsx

```diff
@@ -238,6 +253,9 @@ class VerifyActivities extends Component {
             stats={stats}
           />
         </aside>
+        {
```

Comment:
I commented on this on another PR. I am a big fan of computing this in the render i.e

let messageHtml = "";
if(message){
messageHtml = <SnackBar message={message} />
};

and then placing the `{messageHtml}` in the JSX. does that makes sense?

Reply:
I guess this does help with reusability I will change this

Reply:
Sorry @kimani-ndegwa  can you give me some context on why you think this is better so I can learn as well 👍

Reply:
Sure thing. It is more of a best practice because it allows for a bit more readability. Plus we stick to allowing the JSX to render actual tags instead of putting conditionals within the JSX. Imagine a situation where you have more than one condition that includes whether we receive data from the backend, whether we have a list of items that have evaluated to a particular length. The return statement becomes exponentially bigger. The pattern I am proposing helps keep the return concise and makes it easier to understand the logic.Makes sense or too much?

Reply:
Also, I would like to add @kimani-ndegwa after investigating the ternary I initially defined will render a 0 due to it being a falsy and Javascript coercing it to a number.

Reply:
@kimani-ndegwa  I have resolved this.